example/classifier/Makefile.am

```diff
@@ -24,8 +24,6 @@ all-local:
 			fi \
 		done \
 	fi
-	ln -f -s $(top_srcdir)/platform/$(with_platform)/test/example/classifier/pktio_env \
-		pktio_env
```

Comment:
If this was `$(top_builddir)` instead of `$(top_srcdir)`, would it work then? Or just `../../platform/$(with_platform)/test/example/classifier/pktio_env`, like in example/ipsec_*